deployments/scroll/usdc/configuration.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "Compound USDC",
+  "symbol": "cUSDCv3",
+  "baseToken": "USDC",
+  "baseTokenAddress": "0x06eFdBFf2a14a7c8E15944D1F4A48F9F95F663A4",
+  "baseTokenPriceFeed": "0x43d12Fb3AfCAd5347fA764EeAB105478337b7200",
+  "borrowMin": "0.001e6",
+  "storeFrontPriceFactor": 0.6,
+  "targetReserves": "1000000e6",
+  "rates": {
+    "supplyKink": 0.85,
+    "supplySlopeLow": 0.048,
+    "supplySlopeHigh": 1.6,
+    "supplyBase": 0,
+    "borrowKink": 0.85,
+    "borrowSlopeLow": 0.053,
+    "borrowSlopeHigh": 1.8,
+    "borrowBase": 0.015
+  },
+  "tracking": {
+    "indexScale": "1e15",
+    "baseSupplySpeed": "0.000402083333333e15",
+    "baseBorrowSpeed": "0.000402083333333e15",
```

Comment:
These are the COMP speeds that need to be decided by Gauntlet still.

Reply:
set to 0 as Gauntlet is not recommending rewards for now